test/collins/util/parsers/LshwParserSpec.scala

```diff
@@ -492,5 +492,18 @@ class LshwParserSpec extends mutable.Specification {
       }
     } // wonky opterons
 
+    "Parse GPU server" in {
```

Comment:
I am not surprised. LSHW is notoriously shitty in that it produces different output between versions of software and hardware platforms. Do you have a stack trace of the failed tests, or somewhere to begin digging in? I would be ok with a manually edited XML blob if it makes the tests pass (i.e. gut any lshw sections that are making the test explode as an MVP)